node/node.go

```diff
@@ -552,7 +559,7 @@ func (node *Node) Start() error {
 					errChan <- withError{
```

Comment:
Shall we make errChan a buffered channel? Else this step can be blocked and the sample log behavior might effect message handling/

Reply:
ok, will make it 100.